.chloggen/stanley.liu_add-metrics-client.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: exporter/datadogexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add trace agent metrics client in order to report trace agent telemetry metrics
```

The changelog should describe user-facing changes

```suggestion
note: Internal telemetry metrics for the Datadog traces exporter are now reported through the Collector's self-telemetry 
```